stress-example/image_search/flows/query.yml

```diff
@@ -17,7 +17,7 @@ pods:
     polling: all
     uses_after: _merge_matches
     timeout_ready: 100000 # larger timeout as in query time will read all the data
-    volumes: './workspace:/workspace/workspace'
+    volumes: './workspace:/docker_workspace/workspace'
```

Comment:
can u try simply `/docker-wokspace` without the extra `workspace`?

Reply:
But how is this going to work? how is it going to find it? If I try it like that doesn't get any data:
`vec_idx/2/ContainerRuntime@34340[I]:num_dim extracted from 'ref_indexer' to None`